frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -69,13 +70,18 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     handlePromise,
     close,
     cancel,
-    allowedBusses = new Set(DiskBus.getAll()),
+    templateValidations,
   } = props;
   const asId = prefixedID.bind(null, 'disk');
   const disk = props.disk || DiskWrapper.EMPTY;
   const volume = props.volume || VolumeWrapper.EMPTY;
   const dataVolume = props.dataVolume || DataVolumeWrapper.EMPTY;
-  const validAllowedBusses = allowedBusses || new Set(DiskBus.getAll());
+  const validAllowedBusses = !templateValidations
+    ? new Set(DiskBus.getAll())
+    : templateValidations.getAllowedBusses();
+  const recommendedBusses = !templateValidations
```

Comment:
we should not label buses with `--- Not Recommended ---` if `templateValidations.getRecommendedBusses()` has no intersection with validAllowedBusses as it just clutter the UI and does bring anything useful for the user

Reply:
It doesn't make sense, a recommended bus must be a valid one - so there will always be an intersection of at least one bus type.

Reply:
Where does it say so? The user can still do it right?
IMO we should guard against that.

Reply:
I think it's common sense. but if you wish to remove the label I don't mind. we can always add it in the future.

Reply:
I am not saying we should remove this feature. Let's keep it 

I am just saying we should not label it this way in this edge case.
example:

valid: `['Virtio']`
recommended: `[ 'sata' ]`


as it does not make sense to show the label, because there is not anything we could recommend